### ISSUE TITLE
import not found 

### ISSUE THREAD

**Issue by vvsuperman** [rgalyeon/Scroll_V2#2]
in all the module , util can not import, it shoud add .. infront of utils

import aiohttp
from loguru import logger

from ..config import NFT_ORIGINS_CONTRACT, NFT_ORIGINS_ABI
from ..utils.gas_checker import check_gas
from ..utils.helpers import retry
from .account import Account

class NftOrigins(Account):

**Comment by rgalyeon** [comment]
It depends on how you run the code. If you run the code from the project root folder, you don't need "..". Go to the project folder and run main.py. If you run it in pycharm, check the script startup settings.

**Comment by vvsuperman** [comment]
yeah, I integrated with django then put the code in one module, then the problem appeared. is there any method to avoid the whole change of the source code?
